controllers/topics.js

```diff
@@ -274,16 +276,12 @@ router.post('/', async (req, res, next) => {
 
 	api(req).post('/lessons/', {
 		json: data, // TODO: sanitize
-	}).then(() => {
+	}).then((lesson) => {
 		if (req.body.referrer) {
 			res.redirect(`${(req.headers.origin)}/${req.body.referrer}`);
 		}
-		res.redirect(
-			context === 'courses'
-				? `/courses/${req.params.courseId
-				}${req.query.courseGroup ? `/groups/${req.query.courseGroup}` : '/?activeTab=topics'}`
-				: `/teams/${req.params.teamId}/?activeTab=topics`,
```

Comment:
@SevenWaysDP Aus meiner Sicht gibt es keine Möglichkeit mehr in Teams eine Lesson zu erstellen. Deswegen entfällt die Möglichkeit hier. Courses und Coursegroups sind beide durch die Lösung abgedeckt.